packages/block-library/src/navigation/index.php

```diff
@@ -1457,17 +1457,24 @@ function block_core_navigation_set_ignored_hooked_blocks_metadata( $inner_blocks
 /**
  * Updates the post meta with the list of ignored hooked blocks when the navigation is created or updated via the REST API.
  *
+ * @since 6.5.0
+ * @access private
  * @param stdClass $post Post object.
+ * @return stdClass
```

Comment:
```suggestion
 * @access private
 * @since 6.5.0
 *
 * @param stdClass $post Post object.
 * @return stdClass The updated post object.
```

In line with PHP docs standards.